src/Model/Table/PanelsTable.php

```diff
@@ -56,7 +56,7 @@ public function initialize(array $config)
     public function findByRequest(Query $query, array $options)
     {
         if (empty($options['requestId'])) {
-            throw new \RuntimeException('Missing request id in findByRequest.');
+            throw new \RuntimeException(__d('debug_kit', 'Missing request id in findByRequest.'));
```

Comment:
findByRequest should be findByRequest()
and it should not be part of the translation string!